spec/controllers/rentals_controller_spec.rb

```diff
@@ -75,6 +75,33 @@
       get :new
       expect(response).to render_template :new
     end
+
+    context 'assiging users for search' do
+      before do
+        User.destroy_all
+        @dept_one = create :department
+        @dept_one_users = create_list :user, 10, department: @dept_one
+        @other_users = create_list :user, 10 # not in @dept_one
+      end
+
+      it 'assigns all users if that user has the permission' do
+        u = @other_users.first
+        g = create(:group)
+        g.permissions << create(:permission, controller: 'rentals', action: 'assign_anyone')
```

Comment:
Side note: We should create a function called create_with_permission/s inside group.